data/signature_gene_network.txt

```diff
@@ -1,102 +1,102 @@
-Sig1pos	PA0039	PA0139	PA0376	PA0409	PA0456	PA0576	PA0579	PA0762	PA0805	PA0865	PA0973	PA0996	PA1000	PA1159	PA1288	PA1544	PA1579	PA1582	PA1583	PA1588	PA1592	PA1777	PA1800	PA1801	PA1802	PA1804	PA1830	PA1847	PA2623	PA2634	PA2741	PA2742	PA2743	PA2853	PA2966	PA3031	PA3049	PA3162	PA3201	PA3202	PA3244	PA3385	PA3529	PA3533	PA3637	PA3655	PA3656	PA3745	PA3814	PA3815	PA3819	PA4067	PA4239	PA4241	PA4242	PA4243	PA4244	PA4246	PA4247	PA4248	PA4249	PA4250	PA4251	PA4253	PA4254	PA4257	PA4258	PA4259	PA4262	PA4263	PA4264	PA4265	PA4266	PA4267	PA4268	PA4272	PA4273	PA4315	PA4366	PA4433	PA4463	PA4464	PA4563	PA4568	PA4578	PA4661	PA4671	PA4741	PA4745	PA4761	PA4922	PA4942	PA4944	PA5013	PA5148	PA5178	PA5240	PA5257	PA5276	PA5300	PA5301	PA5316	PA5337	PA5348	PA5438	PA5461	PA5491	PA5555	PA5558	PA5559	PA5560	PA5569	PA5570
```

Comment:
Also here, how was this file modified?

Reply:
I used `cut` command to copy the file w/o the first command, then created a new column with a simple Python script, then used `paste` command to clue these two parts together.

Reply:
The procedure is something like this:
```
cut -f2- signature_gene_network.txt > data_wo_col1.txt
# use python to generate a new column with filename "new_col1.txt" ...
paste new_col1.txt data_wo_col1.txt > signature_gene_network.txt
```
In `cut/paste` commands, the default delimiter is tab. (You can customize it of course.)

Reply:
Ok, gotcha. I was wondering if we should tell the users how we made these modifications for these files, but I guess that is up to them, as long as they know that we have modified these files.